src/_config.ts

```diff
@@ -15,14 +15,21 @@ export const pluginsThatGoAtTheEnd = new Set([
  * Return the configuration for the @semantic-release/git plugin to commit
  * changes in the specified assets
  * @param assets List of assets to commit
+ * @param requireCI to controll when commit requires ci check
  */
-export function semanticReleaseGit(assets: string[]): SemanticReleasePlugin {
+export function semanticReleaseGit(
+  assets: string[],
+  requireCI = false
+): SemanticReleasePlugin {
+  const message = requireCI
+    ? "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
```

Comment:
This should still be `ci`. CI commits don't trigger a new release. `chore` is deprecated as a commit type

Reply:
done ✅